docs-src/cli/cmd-options/set-as-default.md

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
Add to `cmd-options.md`. (Apologies for missing this one during my previous review.)